lib/grape/validations/validate_service.rb

```diff
@@ -0,0 +1,51 @@
+module Grape
+  module Validations
+    class ValidateService
```

Comment:
I think service is very ambitious :) Maybe `ValidateService` should be just `Validator`, what do you think?